src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+
+    if (secondCell && lastCell) {
+      const clonedCell = secondCell.cloneNode(true);
+
+      row.insertBefore(clonedCell, lastCell);
+    }
+  });
+}
```

Comment:
It's a good practice to add a brief comment above each function to describe what it does. This will make your code easier to understand for other developers.